src/litdata/streaming/item_loader.py

```diff
@@ -409,11 +409,20 @@ def delete(self, chunk_index: int, chunk_filepath: str) -> None:
 
     def close(self, chunk_index: int) -> None:
```

Comment:
Let's move this logic to delete instead. I think one worker closes the memap while another reads from it

Reply:
Yes, you are right ! Might be something else.